lib/src/scope.dart

```diff
@@ -46,7 +46,7 @@ class Scope {
   /// Calls an isolate-scoped RPC named [method] with [params].
   ///
   /// This always adds the `isolateId` parameter to the RPC.
-  Future<Object> sendRequest(String method, [Map<String, Object> params]) async {
+  Future<dynamic> sendRequest(String method, [Map<String, Object> params]) async {
```

Comment:
Are you suggesting to use the "as Map<String, dynamic>" in all the references mentioned above? is there a danger to just replace Object -> dynamic?